src/consensus/consensus.h

```diff
@@ -15,7 +15,7 @@ static const unsigned int BIP102_FORK_BUFFER = (144 * 90);
 /** The maximum allowed size for a serialized block, in bytes (only for buffer size limits) */
 static const unsigned int MAX_BLOCK_SERIALIZED_SIZE = (8 * 1000 * 1000);
 /** The maximum allowed weight for a block, see BIP 141 (network rule) */
-static const unsigned int MAX_BLOCK_WEIGHT = 4000000;
+static const unsigned int MAX_BLOCK_WEIGHT = (8 * 1000 * 1000);
```

Comment:
To tell you the truth, I'm not sure what effect, if any, the MAX_BLOCK_WEIGHT and MAX_BLOCK_SERIALIZED_SIZE would have if they're each raised to 8M with the softfork. I considered the fact that they may be clamped in practice, but I've never tested BIP141 with higher weight limits. It's just something to consider.